### ISSUE TITLE
Berserker's Rage

### ISSUE THREAD

**Issue by nstevens26** [Cards-of-Chaos/Game-Design#133]
You body begins to swell with chaos energy throwing you into a fit of rage. You gain 15 armor points and all enemies with in a 7 space radius of you must make a spell saving throw. On a failed save, the enemy deals only half damage to all allies except yourself for 2 turns.

Duration: 2 Turns.

**Comment by D-D-2** [comment]
Should this be a personality save?

**Comment by nstevens26** [comment]
> Should this be a personality save?

I agree. I wanted to ask you but I think any spell that requires a saving throw should have the type of saving throw assigned to it. If you agree, then we need to go back through all the spells that require a saving throw and add the type

**Comment by D-D-2** [comment]
> > Should this be a personality save?
> 
> 
> 
> I agree. I wanted to ask you but I think any spell that requires a saving throw should have the type of saving throw assigned to it. If you agree, then we need to go back through all the spells that require a saving throw and add the type

Definitely. Let's make that a tag on the spells that require saves. For clarity, the saves are: 
- Personality
- Chaos 
- Power
- Vitality (Name subject to change)
